README.rst

```diff
@@ -26,6 +26,18 @@
     :target: https://mybinder.org/v2/gh/scikit-learn-contrib/hdbscan
     :alt: Launch example notebooks in Binder
 
+=======
+About this fork
+=======
+
+While writing my Bachelor's Thesis which includes a comparison of clustering 
+validity metrics like the one presented in 
+`this paper <https://www.semanticscholar.org/paper/Relative-clustering-validity-criteria%3A-A-overview-Vendramin-Campello/ad93d3b55fb94827c4df45e9fc67c55a7d90d00b>`_ 
+(but focused on the application of metrics on non-spherical/density-based clusters) 
+I noticed that DBCV might actually perform better if it just used euclidean distances 
+instead of mutual reachablity distances when constructing the minimum spanning trees. 
+This fork adds a parameter to the DBCV metric which toggles the usage of mutual reachability distances. 
+(the approach of the paper referenced above is also used in the DBCV paper)
```

Comment:
You will want to remove this from the README (you could possibly add a similar entry to the FAQ in the documentation or something like that.

Reply:
Sure. I sadly don't have access to my dev-machine until Wednesday night/Thursday morning. I will take care of the revert then. Additionally I will look into modifying the documentation as well as having a last look at the naming of "my" variables/functions. Then we can merge this Thursday/Friday